infrastructure/disk.tf

```diff
@@ -114,9 +114,51 @@ resource "aws_s3_bucket" "data_refinery_compendia_bucket" {
   }
 }
 
-resource "aws_s3_bucket_metric" "compendia_metrics" {
+resource "aws_s3_bucket_metric" "compendia_bucket_metrics" {
   bucket = "${aws_s3_bucket.data_refinery_compendia_bucket.bucket}"
-  name   = "data-refinery-compendia-metric-${var.user}-${var.stage}"
+  name   = "EntireBucket"
+}
+
+resource "aws_cloudwatch_event_rule" "compendia_object_metrics" {
+  name = "data-refinery-compendia-object-metric-${var.user}-${var.stage}"
+  description = "Download Compendia Events"
+
+  event_pattern = <<PATTERN
+{
+  "source": [
+    "aws.s3"
+  ],
+  "detail-type": [
+    "AWS API Call via CloudTrail"
+  ],
+  "detail": {
+    "eventSource": [
+      "s3.amazonaws.com"
+    ],
+    "eventName": [
+      "GetObject"
+    ],
+    "requestParameters": {
+      "bucketName": [
+        "data-refinery-s3-compendia-${var.user}-${var.stage}"
```

Comment:
This is the bucket that compendia are stored in? I think this should be `"${aws_s3_bucket.data_refinery_compendia_bucket.bucket}"` (assuming that's the correct bucket).

Reply:
I think that would get the ID/arn no?

Reply:
https://www.terraform.io/docs/providers/aws/r/s3_bucket.html#bucket says:
```
bucket - (Optional, Forces new resource) The name of the bucket. If omitted, Terraform will assign a random, unique name.
```

Reply:
I mean that you should use the variable rather than the value.